### ISSUE TITLE
User profile inputs and textarea have no focus styles

### ISSUE THREAD

**Issue by muffinresearch** [mozilla/addons-frontend#5573]
Using a keyboard to focus the inputs and textarea there are no focus styles.

<img width="665" alt="user_profile_for_muffinresearch__script_alert_0___script__ _add-ons_for_firefox" src="https://user-images.githubusercontent.com/1514/42620964-6187e300-85b4-11e8-8565-b87d99a776db.png">

<img width="1025" alt="user_profile_for_muffinresearch__script_alert_0___script__ _add-ons_for_firefox" src="https://user-images.githubusercontent.com/1514/42621010-7b365b1a-85b4-11e8-96a5-afa8b185478a.png">

**Comment by ioanarusiczki** [comment]
Verified fixed on AMO dev with FF61(Win10 and Android 8.0)

![works](https://user-images.githubusercontent.com/33448286/42810230-1f10f86c-89c0-11e8-9359-da47448d12b4.gif)

I filed https://github.com/mozilla/addons-frontend/issues/5629 which is reproducible only on android platform.
